hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+This patch adds a `.hypothesis` property to invalid test functions, bringing
+them inline with valid tests and fixing a bug where `pytest-asyncio` would
```

Comment:
```suggestion
This patch adds a ``.hypothesis`` property to invalid test functions, bringing
them inline with valid tests and fixing a bug where :pypi:`pytest-asyncio` would
```

RST formatting quirk: double-backticks for monospace, single for various magic.  And we have some of that magic for PyPI links 😁